bpf/bpf_lxc.c

```diff
@@ -440,6 +442,9 @@ static __always_inline int handle_ipv6_from_lxc(struct __ctx_buff *ctx, __u32 *d
 	if (verdict == DROP_POLICY_AUTH_REQUIRED) {
 		if (ct_status == CT_NEW) {
 			ct_state_new.src_sec_id = SECLABEL;
+			/* Don't pass ext_err: errors from ct_create6 are ignored
+			 * anyway, and ext_err is used by __policy_can_access.
+			 */
```

Comment:
That (and equivalent IPv4 code) is the part we will need to update once Jarno's PR is merged, right? Then we can probably reuse the existing issue for to keep track of it.

Reply:
I didn't check Jarno's changes in detail. It was more of a general comment than on this specific case.

Your approach makes sense to me 👍